### ISSUE TITLE
General registration

### ISSUE THREAD

**Issue by alhirzel** [yuyichao/LibArchive.jl#11]
Is there a reason for this to not be part of the General registry? @yuyichao

**Comment by yuyichao** [comment]
No. Just that I haven't bothered to look up the latest procedure...
